src/main.js

```
@@ -129,11 +129,42 @@ function ping(event) {
 }
 addEventListener("pagehide", ping);
 addEventListener("visibilitychange", ping);
-addEventListener("DOMContentLoaded", function() {
-  webVitals.getCLS(sendToGoogleAnalytics);
-  webVitals.getFID(sendToGoogleAnalytics);
-  webVitals.getLCP(sendToGoogleAnalytics);
-});
+
+/**
+ * Injects a script into document.head
+ * @param {string} src path of script to be injected in <head>
+ * @return {Promise} Promise object that resolves on script load event
+ */
+const dynamicScriptInject = (src) => {
+  return new Promise((resolve, reject) => {
+    const script = document.createElement("script");
+    script.src = src;
+    script.type = "text/javascript";
+    document.head.appendChild(script);
+    script.addEventListener("load", () => {
+      resolve(script);
+    });
+    script.addEventListener("error", (event) => { 
```

I think it is fine to ignore the error case and just let it go to the console.